src/Microsoft.Build.Sql.Templates/sqlproject/.gitignore

```diff
@@ -0,0 +1,484 @@
+## Ignore Visual Studio temporary files, build results, and

```

Comment:
Totally get that we needed to use this approach for now, but I'm watching https://github.com/dotnet/templating/issues/2771 in hopes that it gets implemented one day, so that we can remove this checked-in version of the .gitignore file 😄.